### ISSUE TITLE
documentation & "Getting Started" in SimPEG

### ISSUE THREAD

**Issue by lheagy** [simpeg/simpeg#897]
Right now, questions around conventions for 
- coordinate system
- fourier transform 
are not necessarily obvious in the docs. I think we should have these in the "Getting Started" section. 

My suggestion for the "Getting Started" heading of SimPEG is similar to what we have in the [discretize docs](http://discretize.simpeg.xyz/), what do folks think of the following for the getting started heading

- **Why SimPEG**: current content on that page
- **Installing**: contains instructions for conda / pip installation as well as developer installation)
- **Quick overview**: conventions on coordinate system, forward simulations (what can we solve & how [e.g. FV, integral equation, ...]), inverse problem (frame as an optimization with data misfit + regularization). _or maybe this should be on the index page??_
- **Gallery / Examples**: as is

**Comment by bluetyson** [comment]
Yes, I think this could use some expansion.  The 'this' then 'this' then 'this' part of using the code is reasonably clear.

Your tutorial here is excellent :- https://github.com/simpeg/transform-2020-simpeg/blob/master/1-century-dcip-inversion.ipynb (and presumably took lots of work).  One of the best explanations of something highly technical with code I think I have seen.

Like most programming stuff, we skip over what is going on to the programming parts generally.

Note that I am obviously not a geoscientist.  So I look at it from that point of view.  Things I have considered after running a few tests:

http://docs.simpeg.xyz/content/tutorials/03-gravity/plot_inv_1a_gravity_anomaly.html#sphx-glr-content-tutorials-03-gravity-plot-inv-1a-gravity-anomaly-py

It is really good there is an automatic example that lets a first time user so things running, so well done.

etc,

xyz_topo = np.loadtxt(str(topo_filename))

# Load field data
dobs = np.loadtxt(str(data_filename))

# Define receiver locations and observed data
receiver_locations = dobs[:, 0:3]
dobs = dobs[:, -1]

date-of-births maybe not the best variable name

and nice to have some plots etc.

However, could explain briefly 'this data is in each file, in these units' and what sort of gravity anomaly in what units?  
What are the coordinates, what do they have to be? Metres?  Latitude, etc?

Starts talking about density contrasts - why?  Given the example is titled Gravity Anomaly Data. Completely obvious to a geophysicist, yes.

Could be a couple of sentences saying background is this, if instead we make it all 2.67... and survey data, what should it look like to be useful?  How much do you need?  How big an example will run on an off the shelf PC?
If you only have 11 points and they are all around -0.25 or something, what happens?

Plotting models is good, but the more esoteric parts

ax1 = fig.add_axes([0.1, 0.1, 0.73, 0.8])
mesh.plotSlice(
    plotting_map * recovered_model,
    normal="Y",
    ax=ax1,
    ind=int(mesh.nCy / 2),
    grid=True,
    clim=(np.min(recovered_model), np.max(recovered_model)),
    pcolorOpts={"cmap": "viridis"},
)
ax1.set_title("Model slice at y = 0 m")

etc.

How this suddenly comes out s a y=0 slice?  How about y = 25 or x = 0?

Also I imagine if this is done once, then will be reuseable on a bunch of examples a lot of the writing.

That being said, I am really impressed an example like that just worked.  And documentation isn't fun.

**Comment by bluetyson** [comment]
Some of which might be in the aforementioned 'conventions on coordinate systems' etc. So write, once, add link. :)
